api/provider_api.md

```diff
@@ -1,6 +1,6 @@
 <!--:nav_order:2-->
 
-# Adobe I/O Events Provider API
+# Adobe I/O Events Management API
```

Comment:
The title of the page is wrong, this page talks about Providers API, so it shouldn't be named Management API

Reply:
done https://github.com/AdobeDocs/adobeio-events/pull/77/commits/067dbf5c96179cf4f2b7272b0349889c5b8b1b10